info.md

```diff
@@ -0,0 +1,259 @@
+
+**We heartily welcome any and all contributions that match our engineering standards! :raised_hands:**
+
+That being said, this codebase isn't your typical open source project because it's not a library or package with a limited scope—it's our entire product.
+
+## Rules
+
+- **No `console.log`s in any file**: We use the `debug` module across the codebase to log debugging information in development only. Never commit a file that contains a `console.log` as CI will fail your build. The only exceptions are errors, which you can log, but you have to use `console.error` to be explicit about it
+- **Code reviews**: All submissions, including submissions by project members, require review. We use GitHub pull requests for this purpose.
+
+### Contributions and discussion guidelines
+
+🎊All conversations and communities on Dummygram agree to GitHub's [Community Guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines) and [Acceptable Use Policies](https://help.github.com/en/github/site-policy/github-acceptable-use-policies).
+
+🎊Code of conduct is applied for all conversations that happen within the contributor community here on GitHub.
+
+🎊Discussions are important,productive and respectful.
+
+🎊**Remember**:-There are real people on the other side of that screen:exclamation:
+
+### Reporting a bug or discussing a feature idea
+
+If you found a technical bug on Dummygram or have ideas for features we should implement, the issue tracker is the best place to share your ideas. Make sure to follow the issue template and you should be golden! ([click here to open a new issue](https://github.com/narayan954/dummygram/issues/new))
+
+### Fixing a bug or implementing a new feature
+
+- If you find a bug on Dummygram and open a PR that fixes it we'll review it as soon as possible to ensure it matches our engineering standards.
+- If you want to implement a new feature, open an issue first to discuss what it'd look like .
+- If you want to contribute but are unsure to start, we have [a "good first issue" label](https://github.com/narayan954/dummygram/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) which is applied to newcomer-friendly issues. Take a look at [the full list of good first issues](https://github.com/narayan954/dummygram/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and pick something you like!
+- Want to fix a bug or implement an agreed-upon feature? Great, jump to the [local setup instructions](#first-time-setup)!
+
+## Code of Conduct
+
+Dummygram is now part of GitHub. For code of conduct, please see [GitHub's Community Guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines) and [Acceptable Use Policies](https://help.github.com/en/github/site-policy/github-acceptable-use-policies).
+
+<div  align="center"><img  height="70px"  src="https://user-images.githubusercontent.com/77617189/192942891-31b9152c-918b-4fac-af05-0ad6b1f594aa.svg"  /></div>
+
```

Comment:
readme-hindi is untouched, that doesn't make sense. You may remove readme-hindi

Reply:
I followed the basic convention by renaming file names in smallcase letters.

Reply:
I am not talking about naming conventions... I was telling to either remove the file or edit it as well. We can't have different translations in different formats. Doesn't make sense